assets/styles/app.css

```diff
@@ -0,0 +1,324 @@
+  width: 100%;
+  padding: 10%;
+}
+
+.placeholder-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.placeholder {
+  max-height: 250px;
+  width: 100%;
+  object-fit: cover;
+  object-position: center;
+  border-radius: 12px 12px 0 0;
+}
+
+.text-container {
+  border: 1px solid #ebf0ee;
+  width: 100%;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 12px;
+  margin-top: -10px;
+  background: white;
+}
+
+h1 {
+  text-align: center;
+  color: #3a4a42;
+  padding-top: 15px;
+}
+
+.button1 {
+  padding-bottom: 36px;
+}
+
+.button-text {
+  outline: none;
+  border: none;
+  background: #36b37f;
+  color: #ebf0ee;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  font-size: medium;
+}
+
+.text-about {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  grid: 1.5rem;
+}
+
+.text {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.plane {
+  position: absolute;
+  top: -30px;
+  right: 0;
+}
+
+h2 {
+  text-align: center;
+  color: #172b4d;
+  padding: 20px;
+  font-size: 40px;
+}
+
+h6 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+}
+
+.square {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.button2 {
+  display: flex;
+  justify-content: center;
+  padding: 12px;
+}
+
+.text-aline {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-direction: column;
+  padding: 12px;
+  text-align: center;
+}
+
+.plane-image {
+  position: absolute;
+  width: 100%;
+}
+
+.start {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  padding: 55px 0;
+}
+
+.write {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.Language {
+  display: flex;
+  justify-content: center;
+  color: #344563;
+  font-size: xx-large;
+  margin-top: -28px;
+}
+
+.media {
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  list-style-type: none;
+  color: #36b37f;
+  flex-wrap: wrap;
+}
+
+.media li {
+  background: white;
+  padding: 12px;
+  border-radius: 8px;
+  font-size: 12px;
+}
+
+.text-container ul {
+  list-style-type: none;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 13px;
+  gap: 0.5rem;
+  padding: 20px 0;
+}
+
+.text-container ul li {
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 5px;
+}
+
+.final {
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  border-radius: 12px;
+  margin: 0 25px 1px 25px;
+}
+
+.final-place {
+  width: 100%;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 24px;
+}
+
+.final-place img {
```

Comment:
[OPTIONAL] As requested by the previous reviewer please use descriptive names 😉 because this help you keep your code organized and easy to read for other developers and you also in the future.